core/templates/dev/head/domain/skill/ConceptCardBackendApiServiceSpec.ts

```diff
@@ -51,14 +56,16 @@ describe('Concept card backend API service', function() {
       var successHandler = jasmine.createSpy('success');
       var failHandler = jasmine.createSpy('fail');
```

Comment:
Make two tests - fetching concept card for one skill, and fetching for multiple skills

Reply:
Done.